cmake/AMR-WindConfig.cmake.in

```diff
@@ -14,6 +14,9 @@ endif()
 
 find_package(AMReX QUIET REQUIRED)
 
+set(AMReX_HYDRO_DIR "${CMAKE_CURRENT_LIST_DIR}/../AMReX-Hydro")
```

Comment:
Why do we need this? How will this work if we want to use an external `AMReX-hydro` install?

Reply:
Remove this line as we don't want the following invocation to fail. 

```
find_package(AMR-Wind REQUIRED)
```

Reply:
@sayerhs fixed it here https://github.com/Exawind/amr-wind/pull/551/commits/757c13a898e30702325e4b4f4182bc08be4bc0e6